internal/mode/static/policies/observability/generator.go

```diff
@@ -0,0 +1,77 @@
+package observability
+
+import (
+	"fmt"
+	"strings"
+	"text/template"
+
+	ngfAPI "github.com/nginxinc/nginx-gateway-fabric/apis/v1alpha1"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/framework/helpers"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/policies"
+)
+
+var tmpl = template.Must(template.New("observability policy").Parse(observabilityTemplate))
+
+const observabilityTemplate = `
+{{- if .Tracing }}
+otel_trace {{ .Strategy }};
+  {{- if .Tracing.Context }}
+otel_trace_context {{ .Tracing.Context }};
+  {{- end }}
+  {{- if .Tracing.SpanName }}
+otel_span_name "{{ .Tracing.SpanName }}";
+  {{- end }}
+  {{- range $attr := .Tracing.SpanAttributes }}
+otel_span_attr "{{ $attr.Key }}" "{{ $attr.Value }}";
+  {{- end }}
+  {{- range $attr := .GlobalSpanAttributes }}
+otel_span_attr "{{ $attr.Key }}" "{{ $attr.Value }}";
```

Comment:
![Screenshot 2024-05-23 at 2 55 03 PM](https://github.com/nginxinc/nginx-gateway-fabric/assets/8921145/6fa9d08d-5eb1-481f-a5d0-2d724bb01081)
 It works, but (at least with Jaeger) a Warning is displayed. This may be preferred behavior.